src/interp/istream.h

```diff
@@ -98,7 +98,7 @@ class Istream {
   //
   // Each opcode is a SerializedOpcode, and each immediate is a u32.
   static const Offset kBrTableEntrySize =
-      sizeof(SerializedOpcode) * 2 + 3 * sizeof(u32);
+      sizeof(SerializedOpcode) * 3 + 4 * sizeof(u32);
```

Comment:
I'm having trouble understanding how the above comment describes these magic numbers.    Does the comment need updating?  If not, why not?

Reply:
Thanks yes, the part that needs updating is the "made of two instructions" part. It's now three instructions so I'll just add the missing bit to the comment.